lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Transactions.hs

```diff
@@ -1739,13 +1739,27 @@ spec = describe "SHELLEY_TRANSACTIONS" $ do
     -- +---+----------+----------+------------+--------------+
     it "TRANS_LIST_02,03x - Can limit/order results with start, end and order"
         $ \ctx -> runResourceT $ do
-        let a1 = Quantity $ sum $ replicate 10 minUTxOValue
-        let a2 = Quantity $ sum $ replicate 10 (2 * minUTxOValue)
-        w <- fixtureWalletWith @n ctx $ mconcat
-                [ replicate 10 minUTxOValue
-                , replicate 10 (2 * minUTxOValue)
-                ]
-        txs <- listAllTransactions @n ctx w
```

Comment:
Was the problem that the faucet txs would be rolled back? Do you think a `threadDelay` would have helped as well as your solution?

Reply:
👌 👍